starfyre/parser.py

```diff
@@ -295,7 +313,12 @@ def handle_data(self, data):
             )
         )
         
-        self.children.append(wrapper_div_component) 
+        # self.children.append(wrapper_div_component)
+        if len(self.stack) > 0:
+                parent_node = self.stack[-1]
+                parent_node.children.append(wrapper_div_component)
+        else:
+            self.current_children.append(wrapper_div_component) 
```

Comment:
can you please explain this?

Reply:
@SuelenKarbivnychyy 👋 

> else block will just handle the situation when we don't have parent_node, (in case we have just text in our pyml, for example
<pyml> Text </pyml>)


This should never be the case. If we have a block without a parent node then we should raise an `Exception`

Reply:
> @SuelenKarbivnychyy 👋
> 
> > else block will just handle the situation when we don't have parent_node, (in case we have just text in our pyml, for example
> >  Text )
> 
> This should never be the case. If we have a block without a parent node then we should raise an `Exception`

Yes. I addressed it on the latest PR